test/config_tests.ml

```diff
@@ -1,16 +1,36 @@
+let string_of_file filename =
+    let ch = open_in ( ("sample-configuration-files/" ^ filename) ) in
+    let s = really_input_string ch (in_channel_length ch) in
+    close_in ch;
+    s
+
 let pmsg =
   Alcotest.testable (fun ppf (`Msg s) -> Fmt.pf ppf "Error @[<v>(%s)@]" s)
     (fun (`Msg a) (`Msg b) -> String.equal a b)
 
 let conf_map = Alcotest.testable
     Openvpn.Config.pp Openvpn.Config.(equal eq)
 
-let parse_noextern conf =
+let parse_noextern_client conf =
   Openvpn.Config.parse_client ~string_of_file:(fun path ->
       Rresult.R.error_msgf
         "this test suite does not read external files, \
          but a config asked for: %S" path) conf
 
+let parse_noextern_server conf =
+  Openvpn.Config.parse_server ~string_of_file:(fun path ->
+      Rresult.R.error_msgf
+        "this test suite does not read external files, \
+         but a config asked for: %S" path) conf
+
+let ok_cert_userpass_client () =
+  let conf = string_of_file "client-tcp-certauth-passauth.cfg" in
+  let parsed = parse_noextern_client conf in
+  ignore parsed;
+  (*FIXME*)
+  Alcotest.(check bool) "cert_userpass works" true true
```

Comment:
I'm adding a dummy comment here we can resolve when this is implemented.

Reply:
removed it for now